cmd/payload-testing-ui/server.go

```diff
@@ -225,6 +241,27 @@ func (s *server) runDetails(w http.ResponseWriter, r *http.Request) {
 			ret := fmt.Sprintf(`<a href="https://github.com/%s/%s/commit/%s">%s</a>`, org, repo, h, h)
 			return template.HTML(ret)
 		},
+		"commentLink": func(obj *metav1.ObjectMeta, pr *prpqv1.PullRequest, config *prpqv1.ReleaseControllerConfig) template.HTML {
+			created := obj.GetCreationTimestamp().Format("Jan 2, 2006 15:04")
+			author := template.HTMLEscapeString(pr.Author)
```

Comment:
We cannot do this yet, this is the author of the pull request, not the comment.

Reply:
It's just text at the moment. I know I have to implement the comment.

Reply:
I will remove it for now.